module3/projects/consultancy/inception.md

```diff
@@ -0,0 +1,184 @@
+---
```

Comment:
This page is a little messy with a mix of call out boxes and drop downs. I think just giving it a once-over to streamline the formatting would be helpful.

Reply:
I tried to do dropdowns for all the sections, but any ## headers looked weird with dropdowns, so I kept those as is and just put the specific components of each of those inception sections in a dropdown. Not sure how else to organize it so if you have ideas, I'm happy to try them out!

Reply:
Nevermind I missed your overall comment! Will try to play around with that. I think I'm also not intentionally making callout boxes but if I use a `##` header it's happening? Going to remove those